pkgs/development/misc/avr/gcc/wrapper.nix

```diff
@@ -0,0 +1,21 @@
+# Setup paths needed by avrgcc-unwrapped
+{ stdenv, writeText, makeWrapper, avrgcc-unwrapped, avrlibc }:
+stdenv.mkDerivation rec {
+  name = "avr-gcc-wrapper-${version}";
+  version = stdenv.lib.strings.getVersion avrgcc-unwrapped.name;
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ avrgcc-unwrapped avrlibc ];
+  builder = writeText "builder.sh" ''
```

Comment:
This can be simplified by using `buildCommand` (see https://github.com/NixOS/nixpkgs/issues/48205#issuecomment-429419972)